indexer/services/ender/src/helpers/kafka-helper.ts

```diff
@@ -254,6 +257,7 @@ export function generateTransferContents(
       createdAtHeight: transfer.createdAtHeight,
       transactionHash: transfer.transactionHash,
     },
+    blockHeight,
```

Comment:
this doesn't need to be a key value?

Reply:
it is a key value. it's short hand annotation.

Reply:
it's a minor nit, but might be better to explicitly specify the key and value to match the style of the rest of the map

Reply:
Rest of the map is structured that way because the value name is different from the key name. In our code base, when key=value, we use this short hand annotation as convention. See for example `generateFillSubaccountMessage`